<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>554</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="deleteitembutton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 0, 0);
color:rgb(255, 255, 255);
font-size:13pt</string>
           </property>
           <property name="text">
            <string>Delete Item</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addcolumnbutton">
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(85, 170, 0);
font-size:15pt;</string>
           </property>
           <property name="text">
            <string>Add Column</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" rowspan="4">
        <layout class="QHBoxLayout" name="hrlayout"/>
       </item>
       <item row="0" column="0" rowspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="boardnamelabel">
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 127);
font-size:30pt;</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QPushButton" name="btmp_button">
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
font-size:13pt;
background-color:rgb(167, 168, 167);</string>
           </property>
           <property name="text">
            <string>Back to Mainpage</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QPushButton" name="calbutton">
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
font-size:10pt;
background-color:rgb(0, 0, 127)</string>
           </property>
           <property name="text">
            <string>Calculator</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <property name="styleSheet">
              <string notr="true">background-color:white;
color:black;</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QPushButton" name="pushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
background-color: green;</string>
             </property>
             <property name="text">
              <string>add member</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QPushButton" name="pushButton_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>9</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
background-color: red;</string>
             </property>
             <property name="text">
              <string>remove member</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item alignment="Qt::AlignRight|Qt::AlignTop">
          <widget class="QListWidget" name="memberlist">
           <property name="maximumSize">
            <size>
             <width>508</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
font-size:15pt;
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
